scripts/test_stable_diffusion.py

```diff
@@ -6,8 +6,14 @@
 model_id = "stabilityai/stable-diffusion-xl-base-1.0"
 pipeline = AutoPipelineForText2Image.from_pretrained(model_id, torch_dtype=torch.float16, variant="fp16", use_safetensors=True)
 
-# Move the pipeline to GPU if available
-device = "cuda" if torch.cuda.is_available() else "cpu"
+# Move the pipeline to GPU if available - also supports Apple mps
+if torch.cuda.is_available():
+    device = "cuda"
+elif torch.backends.mps.is_available:
```

Comment:
this is a function so this is always going to be True (I think its currently also wrong in qwen)